config/vufind/config.ini

```diff
@@ -725,6 +725,9 @@ database          = mysql://root@localhost/vufind
 ;userattribute_value_1 = urn:mace:dir:entitlement:common-lib-terms
 ;userattribute_2       = unscoped-affiliation
 ;userattribute_value_2 = member
+; Set to true when shibboleth is running on the proxy side and so we have to
+; read shibboleth attributes from headers instead of server environment variables
+;proxy                  = true
```

Comment:
Maybe `use_headers = true/false` and add a note that care must be taken when using this option so that the headers cannot be spoofed?